byelaws/article_ii.md

```diff
@@ -0,0 +1,147 @@
+
+c. Assist in maintaining sponsor relationships and community partnerships
+
+d. Assume the duties of the Regional Executor during periods of absence or transition
+
+### Subsection 3: Term of office
+
+A Vice Regional Executor shall serve a term of one (1) year and may serve up to three (3) consecutive terms.
+
+## Section 7: Events Coordinator
+
+### Subsection 1: Establishment
+
+There shall be one or more Events Coordinators responsible for managing community activities and events on behalf of Black Python Devs.
+
+### Subsection 2: Duties and responsibilities
+
+Each Events Coordinator shall:
+
+a. Organize all event logistics, including scheduling, communications, and technical requirements
+
+b. Address and resolve any technical issues arising during events to ensure a seamless experience for attendees
+
+c. Serve as host, co-host, or moderator during events, maintaining a welcoming and engaging atmosphere
+
+d. Facilitate and encourage active participation, discussions, and question-and-answer sessions during events
+
+e. Communicate event details and updates to the community in a timely manner
+
+f. Publish event content on Discord and other community platforms to encourage engagement and feedback
+
+g. Collaborate with organizational partners on event planning and promotion
+
+h. Update partnership links and promotional materials as necessary
+
+i. Coordinate the editing, packaging, and uploading of event content to YouTube.
+
+## Section 8: Conference Sponsorship Coordinator
+
+### Subsection 1: Establishment
+
+There shall be one or more Conference Sponsorship Coordinators, each serving as a regional liaison responsible for managing and coordinating sponsorship activities for Black Python Devs conferences.
+
+### Subsection 2: Duties and responsibilities
+
+Each Conference Sponsorship Coordinator shall:
+
+a. Serve as the primary point of contact for sponsors, addressing all inquiries and concerns
+
+b. Provide regular updates and reports to sponsors regarding the status of their sponsorship and the impact of their contributions
+
+c. Work collaboratively with other team members, including event organizers and marketing personnel, to ensure all sponsorship deliverables are fulfilled
+
+d. Collaborate with the marketing team to promote sponsorship opportunities and maximize sponsor visibility before, during, and after conferences
+
+e. Maintain accurate records of all sponsorship agreements, payments, and sponsor contact information
+
+f. Prepare regular reports on sponsorship revenue, trends, and opportunities for improvement
+
+### Subsection 3: Access and permissions
+
+Each Conference Sponsorship Coordinator shall be granted access to:
+
+a. The Sponsorship@blackpythondevs.com email address;
+
+b. The shared documentation folder containing sponsorship agreements, templates, and contact information
+
+c. Collaboration tools utilized by the Black Python Devs documentation team
```

Comment:
```suggestion
```

Remove as the information is not correct but the establishment of what access and permissions has not been defined.